README.md

```diff
@@ -35,6 +35,7 @@ Editor Plugins
 -	[emacs-markdownfmt](https://github.com/nlamirault/emacs-markdownfmt) for Emacs.
 -	Built-in in Conception.
 -	[markdown-format](https://atom.io/packages/markdown-format) for Atom (deprecated).
+- [vscode-markdownfmt](https://marketplace.visualstudio.com/items?itemName=AnmolSinghJaggi.vscode-markdownfmt#overview) for VS Code.
```

Comment:
While you're making changes here, let's move this item two up, so that it's after emacs-markdownfmt and before Conception-go.

The reason for that is that the Atom plugin is deprecated, while yours is active, so it should be above. Conception-go is also legacy, so it can be placed below your VS Code plugin.

Reply:
HI @dmitshur I know about the formatting issue. I dont why but no matter what I do, it always ends being up formatted this way.
If I open up the Readme to edit, it already seems to be formatted correctly. I am not able to fix this.
Want some help from you on this.

Reply:
> I dont why but no matter what I do, it always ends being up formatted this way.

I suspect your editor is hiding the distinction between tabs and spaces from you, making it hard to make this change. Depending on the editor, it should be possible to configure it to use tabs.

You should be able to use `markdownfmt` itself to format the README.md.

But sure, I can just edit it on my side.